src/H5Fvfd_swmr.c

```diff
@@ -1614,10 +1502,14 @@ H5F__vfd_swmr_update_end_of_tick_and_tick_num(H5F_shared_t *shared,
     FUNC_ENTER_STATIC
 
     /* Get current time in struct timespec */
-    if ( HDclock_gettime(CLOCK_MONOTONIC, &curr) < 0 )
-
-        HGOTO_ERROR(H5E_FILE, H5E_CANTGET, FAIL, \
-                    "can't get time via clock_gettime")
+#ifdef H5_HAVE_WIN32_API
+        if (timespec_get(&curr, TIME_UTC) != TIME_UTC)
+            HGOTO_ERROR(H5E_FILE, H5E_CANTGET, FAIL, "can't get time via timespec_get");
+#else
+        if (HDclock_gettime(CLOCK_MONOTONIC, &curr) < 0) {
+            HGOTO_ERROR(H5E_FILE, H5E_CANTGET, FAIL, "can't get time via clock_gettime");
+        }
+#endif
```

Comment:
This isn't the best protection as clock_gettime(3) isn't (easily) available on some POSIX-y platforms. I'll eventually move this code into a function in H5system.c or H5timer.c when I fix the Windows issues in the tests.